agent/app/agent_capability_unix.go

```diff
@@ -236,3 +237,19 @@ func (agent *ecsAgent) getTaskENIPluginVersionAttribute() (*ecs.Attribute, error
 func defaultIsPlatformExecSupported() (bool, error) {
 	return true, nil
 }
+
+// var to allow mocking for checkNetworkTooling
+var isNetworkToolingAvailable = checkNetworkTooling
+
+// wrapper around exec.LookPath
+var lookPathFunc = exec.LookPath
+
+func checkNetworkTooling() bool {
+	tools := []string{"iptables", "tc"}
+	for _, tool := range tools {
+		if _, err := lookPathFunc(tool); err != nil {
+			return false
```

Comment:
Let's log `err`.

Reply:
Implemented in https://github.com/aws/amazon-ecs-agent/commit/1a3ea5b7f2b55d50e8b41c84c10a75842c262893